stock_picking_filter/models/stock_picking.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2019 C2i Change 2 improve - Eduardo Magdalena <emagdalena@c2i.es>
```

Comment:
Avoid copying a random module to create ours (even if in the draft stage).

Reply:
I understand and fixed it.